roles/openshift_cluster_monitoring_operator/tasks/install.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-cluster-monitoring-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Copy files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/{{ item }}"
+  with_items:
+  - cluster-monitoring-operator.yaml
+  - cluster-monitoring-config.yaml
+
+- name: Copy admin client config
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
+  changed_when: false
+
+- name: Add monitoring project
+  oc_project:
+    state: present
+    name: openshift-monitoring
+    description: Openshift Monitoring
+
+- name: Add cluster admin permission
+  command: >
+    {{ openshift_client_binary }} adm policy add-cluster-role-to-user cluster-admin -z default
```

Comment:
This shouldn't be necessary once securityContexts are all removed... might want to comment here on what remaining images need fixed so we can remove this

Reply:
See https://github.com/openshift/openshift-ansible/pull/7693#discussion_r177876150. There was a massive email thread about this involving @pweil- (SCC SME), I'm not sure the thread reached the entire list. I'll forward.

Reply:
Re: RBAC generally, anything the operator needs permission to do should be explicitly whitelisted in the cluster role bound to the operator's SA rather than a blanket cluster-admin binding.

Reply:
My comment about RBAC (https://github.com/openshift/openshift-ansible/pull/7693#discussion_r177880366) covers this more accurately- I think I confused this line with previous hacks to allow the SA to run as any UID (which is related to SCC). The RBAC comments still stand though; this thing shouldn't have cluster admin.

Reply:
I found the thread, and I understand we need to remove `securityContext` (will send a PR to do that in TPO first thing tomorrow morning) but I still don't understand what does this has to do with cluster roles (unless I'm missing something from that thread).

I'll give it a try tomorrow after I'll remove all `securtiyContext`s (it's a bit late for that on my side of the world right now) and see what happens

Reply:
Sorry, I tried to explain and probably failed. I confused this line with SCC issues. Ignore anything about this cluster role related to SCC. The other comments about the need for this line in terms of RBAC though still require discussion.